Documentation/benchmarks/etcd-2-2-0-benchmarks.md

```diff
@@ -0,0 +1,67 @@
+
+# etcd Cluster
+
+3 etcd 2.2.0 members, each runs on a single machine.
+
+Detailed versions:
+
+```
+etcd Version: 2.2.0
+Git SHA: e4561dd
+Go Version: go1.5
+Go OS/Arch: linux/amd64
+```
+
+# Testing
+
+Bootstrap another machine and use benchmark tool [boom](https://github.com/rakyll/boom) with customized patch to send requests to each etcd member. Check [here](../../hack/benchmark/) for instructions.
+
+The performance is the result of running benchmark script one round. In the testing cluster for this benchmark, the result of multiple rounds could be different up to 20%. An even bigger difference 30% was observed in clusters with the same configuration.
+
+# Performance
+
+## Reading One Single Key
+
+| key size in bytes | number of clients | target etcd server | read QPS | 90th Percentile Latency (ms) |
+|-------------------|-------------------|--------------------|----------|---------------|
+| 64                | 1                 | leader only        | 2816  | 0.4  |
+| 64                | 64                | leader only        | 17572 | 6.0  |
+| 64                | 256               | leader only        | 16943 | 24.1 |
+| 256               | 1                 | leader only        | 2349  | 0.5  |
+| 256               | 64                | leader only        | 16009 | 7.1  |
+| 256               | 256               | leader only        | 15239 | 28.4 |
+| 64                | 64                | all servers        | 49502 | 2.3  |
+| 64                | 256               | all servers        | 50005 | 9.1  |
+| 256               | 64                | all servers        | 48664 | 2.3  |
+| 256               | 256               | all servers        | 49914 | 9.1  |
+
+## Writing One Single Key
+
+| key size in bytes | number of clients | target etcd server | write QPS | 90th Percentile Latency (ms) |
+|-------------------|-------------------|--------------------|-----------|---------------|
+| 64                | 1                 | leader only        | 51   | 22.3 |
+| 64                | 64                | leader only        | 2182 | 31.5  |
+| 64                | 256               | leader only        | 5841 | 52.0 |
+| 256               | 1                 | leader only        | 57   | 20.7 |
+| 256               | 64                | leader only        | 1973 | 38.3 |
+| 256               | 256               | leader only        | 5193 | 59.8 |
+| 64                | 64                | all servers        | 1745 | 54.2  |
+| 64                | 256               | all servers        | 4487 | 78.0  |
+| 256               | 64                | all servers        | 1507 | 59.5  |
+| 256               | 256               | all servers        | 4231 | 82.6  |
+
+## Performance Changes
+
+- Because etcd now records metrics for each API call, read QPS performance is decreased a little in most scenarios. This minimal performance impact was judged a reasonable investment for the breadth of monitoring and debugging information returned.
```

Comment:
Because etcd now records metrics for each API call, read QPS performance sees a minor decrease in most scenarios.

("a little" just isn't the right tone for this location)